### DISCUSSION TITLE
Have a look at my github profile

### DISCUSSION THREAD

**Discussion by Niraj-Dilshan** [skills/.github#122] [Show and tell]
Have a look at my profile
[My Profile](https://github.com/Niraj-Dilshan/Niraj-Dilshan)

**Comment by Kmtengo** [comment]
Amazing profile Niraj :thumbsup:

**Comment by adityakanu** [comment]
Cool, congratulations for Hacktoberfest 2022!

**Comment by The11sun** [comment]
Let me be honest.. I don't really come into this job because that's it...
How do I feel... I was discouraged when I felt the feelings that had
accumulated in my hopes.. If all of you were aware of my reality, I would
understand me better... I felt lonely.. I miss my son.. I've told many
people that since September 5th is my big day. At the moment, I don't have
anyone paying attention and cares about it.. Yes, it means family.. Wife
and child.. Because I was doing something that he didn't know about me.

それは私自身のスティントです。

ในวันที่ ส. 26 พ.ย. 2022 15:17 น. Aditya Kanu ***@***.***>
เขียนว่า:

**Comment by israeldelgado** [comment]
Hey @Niraj-Dilshan , your profile looks sick! Congrats on the great work, hope one day I can make mine look a bit like yours.

A few suggestions :

- Try to make it more professional and appealing by having an excellent professional profile picture

If I were you, I'd remove the 'Buy me a Coffee button, it makes you look a bit needy. Also, it would make the links to your social accounts more relevant and determinant.

Hope you're great bro!

https://github.com/israeldelgado/about-me here's a link to my profile I just created. First couple of days of using Github, I'm just getting the basics and hang of it from the Microsoft course.

**Comment by ManuSquall** [comment]
I pretty like it! It's cool and GG for Hacktoberfest 2022 too 🥳

**Comment by Subslav** [comment]
Hi
